server/src/packages/submission/submission.repository.ts

```diff
@@ -1,16 +1,18 @@
-import { Submission } from './submission.entity';
 import { Option } from 'funfix';
+import { uuidType } from '../../core';
```

Comment:
`core` (or `shared`, or `common` etc) tend to end up being dumping grounds of unrelated things.

If these are to end up as separate libraries, probably worth making it standalone to avoid versioning problems? (Fits the NPM world too?!)

That said, does a UUID object _really_ not exist already?

Reply:
The object exists - just not the type.
Agreed - these can become dumping grounds - that is why we have code reviews :smile:

Reply:
> The object exists - just not the type.

Do you mean the TypeScript declaration? 

> Agreed - these can become dumping grounds - that is why we have code reviews 😄

My first code review comment would be: make this a library. 😛

Reply:
We've agreed this is something we'll do later